File: Programmers/level1/noRepeatNumbers/norepeat_numbers.py
# 문제 출처 : https://programmers.co.kr/learn/courses/30/lessons/12906

def solution(arr):
    answer = []
    for i in range(len(arr)):
        if len(answer) == 0:
            answer.append(arr[i])
        elif answer[-1] == arr[i]:
            continue
        else:
            answer.append(arr[i])
    return answer

# def solution(s):
#     a = []
#     for i in s:
#         if a[-1:] == [i]:
#             continue
#         a.append(i)

#     print(a)
#     return a


# solution([4, 4, 4, 3, 3])

File: Programmers/level1/noRepeatNumbers/noRepeatNumbers.js
// https://programmers.co.kr/learn/courses/30/lessons/12906?language=javascript

export const solution = (arr) => {
    let answer = [];

    for(let i = 0; i < arr.length; i++){
        if(arr[i] === answer[answer.length-1]) continue;
         answer.push(arr[i]);  
    }
    
    return answer;
}